CHANGES.md

```diff
@@ -1,9 +1,138 @@
+
+- [#4178](https://github.com/gap-system/gap/pull/4178) Fixed bugs in `RestrictedPerm` with second argument a range
+
+### Fixed bugs that could lead to crashes
+
+- [#3965](https://github.com/gap-system/gap/pull/3965) Fix potential garbage collector crashes on 64bit ARM systems
+- [#4076](https://github.com/gap-system/gap/pull/4076) Fix an infinite loop in `BoundedRefinementEANormalSeries` if large factors could not be refined, fix protected option of `IsomorphismSimplifiedFpGroup`, improve documentation of `IsAutomorphismGroup`
+
+### Fixed bugs that could lead to error messages
+
+- [#3980](https://github.com/gap-system/gap/pull/3980) Fixed `Gcd` for rational polynomials
+
+### Other fixed bugs
+
+- [#3963](https://github.com/gap-system/gap/pull/3963) Provide automatic compression/decompression of filenames ending `.gz` (as is claimed for example in the documentation of `InputTextFile`)
+- [#3944](https://github.com/gap-system/gap/pull/3944) The error checking in `PartialPerm` has been corrected such that invalid inputs (numbers < 1) are detected
+- [#4006](https://github.com/gap-system/gap/pull/4006) Fix `gac` to ensure that binaries it creates on Linux can load and run, even if a GAP package with a compiled kernel extension (such as `IO`) is present
+
+### Improved and extended functionality
 
-These changes are also listed on the
-[Wiki page](https://github.com/gap-system/GAP/wiki/gap-4.11-release-notes)
+- [#3790](https://github.com/gap-system/gap/pull/3790) Add further information to the library of simple groups
+- [#3840](https://github.com/gap-system/gap/pull/3840) Speed up garbage collection
+
+### Packages
+
+- [#4016](https://github.com/gap-system/gap/pull/4016) Improved `etc/Makefile.gappkg` (used by GAP packages that want to build a simple GAP kernel extension)
+
+### Fixes/improvements in the experimental way to allow 3rd party code to link GAP as a library (libgap)
+
+- [#4081](https://github.com/gap-system/gap/pull/4081) Enhance `GAP_ValueGlobalVariable` to supported automatic variables (see `DeclareAutoreadableVariables`)
+
+### Fixes and improvements for the **Julia** integration
+
+- [#4042](https://github.com/gap-system/gap/pull/4042) Avoid access to JuliaTLS members by using `jl_threadid()` and `jl_get_current_task()` helpers, fix compiler constness warnings in weakptr.c
+- [#4053](https://github.com/gap-system/gap/pull/4053) Fix the logic for scanning tasks in the Julia GC
+- [#4058](https://github.com/gap-system/gap/pull/4058) Refine the logic for scanning Julia stacks
+- [#4071](https://github.com/gap-system/gap/pull/4071) Make the Julia GC threadsafe when used from GAP.jl
+
+### Other changes
+
+- [#3922](https://github.com/gap-system/gap/pull/3922) Build system: New feature to execute `BuildPackages.sh` in parallel mode by adding `--parallel`
+- [#4041](https://github.com/gap-system/gap/pull/4041) Build system: Fix `make check` in out-of-tree builds
+
+### New packages redistributed with GAP
+
+
```

Comment:
According to @alex-konovalov there were no new packages. So either we drop this section ...
```suggestion
```
or we say it, e.g.
```suggestion
### New packages redistributed with GAP

There are no new packages in this GAP release.

```

Reply:
Sorry, I was wrong. There are new packages. Details to follow.

Reply:
So, we had 152 packages in GAP 4.11.0, and 151 in the current merged packages archive, so we are down one package. Looking at https://github.com/homalg-project/homalg_project/issues/221 and the history of revisions of https://github.com/gap-system/gap-distribution/commits/master/DistributionUpdate/PackageUpdate/currentPackageInfoURLList, I've restored this: @mohamed-barakat said we need to remove "PolymakeInterface and Convex from the future GAP distribution. They are now replaced by NormalizInterface and NConvex."  Convex was withdrawn yet in GAP 4.11.0, and now the other package is withdrawn too. In the release notes we can say something like:

"Following the withdrawal of Convex in GAP 4.11.0 because of being superseded by NConvex, the PolymakeInterface had also been withdrawn. These two packages are now replaced by NormalizInterface and NConvex."

(I am not sure of there is a 1-to-1 replacement, or a combination of packages replaces another combination of packages).

Reply:
@alex-konovalov I've uploaded a `package-info.json.gz` retroactively to https://github.com/gap-system/gap/releases/tag/v4.11.0 and compared it to one made in preparation for a new release. I did not see any new packages, but the `PolymakeInterface` package appears to have been removed?

Reply:
The packages `Convex` and `PolymakeInterface` have been deleted some time ago from the `homalg_project` repository as well.